plugins/modules/events_and_notifications_workflow_manager.py

```diff
@@ -1262,12 +1262,15 @@ def get_syslog_destination_in_ccc(self, name):
             return response
 
         except Exception as e:
-            if "Expecting value: line 1 column 1" in str(e):
-                self.log(
-                    "Getting expection as Syslog destination with given name '{0}' not present in Cisco Catalyst"
-                    " Center.".format(name), "WARNING"
-                )
-                return None
+            expected_exception_msg = ["Expecting value: line 1 column 1", "not iterable", "has no attribute"]
```

Comment:
can we write one by one for more readability?

```
expected_exception_msgs = [
    "Expecting value: line 1 column 1",
    "not iterable",
    "has no attribute"
]

```

Reply:
updated.